### DISCUSSION TITLE
ESP32-C3-2424S012 clock variants with touch screen.

### DISCUSSION THREAD

**Discussion by adisorin** [revoxhere/duino-coin#1821] [Ideas]
An idea for @JK-Rolling who implemented oled display in ESP_Code: The same code should be modified for ESP32-C3-2424S012 clock variants with touch screen. I think it's a good idea to see the wallet there being also portable with an external battery.

**Comment by adisorin** [comment]
![20240815_003025](https://github.com/user-attachments/assets/183ce599-ee41-42ff-9f07-dec7a5b18789)

**Comment by adisorin** [comment]
And the conversation must be closed only by the one who opened it!

**Comment by JK-Rolling** [comment]
That is a nice display and revox is the author of ESP_Code who implemented the OLED. 

Anyway, anyone can contribute to ESP_Code as it is open source
